pages/common/ld.md

```diff
@@ -0,0 +1,16 @@
+# ld
+
+> Link object files together.
+> More information: <https://sourceware.org/binutils/docs-2.38/ld.html>.
+
+- Link an object file with no dependencies into the executable:
```

Comment:
```suggestion
- Link a specific object file with no dependencies into an executable:
```

I think it's better to use `a`/`an` here instead of `the` because we are not talking about specific executable. Everything else LGTM. ;)